proposals/README.md

```diff
@@ -1 +1,8 @@
-empty
+# Proposals
+This directory may include contributions from non-IG members.
+Contributors agree to abide by the terms of the W3C Patent and document license policies:
+* [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/)
+    - [Patent Policy FAQ - Q6](https://www.w3.org/2003/12/22-pp-faq.html#non-participants)
+* [W3C Document License Policy](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document)
```

Comment:
IANAL but I wonder if it should be W3C Software and Document Notice and License? [W3C Document License](https://www.w3.org/Consortium/Legal/2015/doc-license) and [W3C Software and Document Notice and License](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document) are different licenses.